docs/configuration/defaults.md

```diff
@@ -1,80 +1,105 @@
 # Defaults
 
-DAG Factory allows you to define Airflow
-[default_args](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/dags.html#default-arguments) and
-additional DAG-level arguments in a `default` block. This block enables you to share common settings and configurations
-across all DAGs in your YAML configuration, with the arguments automatically applied to each DAG defined in the file.
-This is one of DAG Factory's most powerful features; using defaults allows for the dynamic generation of more than a
-single DAG.
+DAG Factory allows you to define default values for DAG-level arguments and Airflow
+[default_args](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/dags.html#default-arguments). There are several ways to accomplish this:
 
-## Benefits of using the default block
+- Define the `default_args` within each DAG definition in the DAGs YAML file;
+- Declare a `default` block within the toplevel of the DAGs YAML file;
+- Define the `default_args_config_dict` argument when instantiating the `DAGFactory` class;
+- Create one or multiple `defaults.yml` and declare the `default_args_config_path` argument in the `DAGFactory` class. This approach includes support for combining multiple `defaults.yml` files.
+
+Although you cannot use the last two configurations together, you can use a combination of the first two configurations with either the third or the last.
+
+Below, we detail how to use each of these approaches and also how to combine them.
+
+## Specifying `default_args` directly in the DAG YAML specification
+
+This configuration affects only the DAG where the `default_args` are defined.
+
+You can override or define specific `default_args` at the individual DAG level. This strategy allows you to customize arguments for each DAG without affecting others. Not only can existing `default_args` be overridden directly in a DAG configuration, but other DAG-level arguments can be added.
```

Comment:
```suggestion
You can override or define specific `default_args` at the individual DAG level. This strategy allows you to customize arguments for each DAG without affecting others. Not only can existing `default_args` be overridden directly in a DAG configuration, but also other DAG-level arguments can be added.
```

Sorry for another quick suggestion, we had the "also" there before, my previous suggestion removed that.